content/security/index.adoc

```diff
@@ -4,48 +4,60 @@ title: "Jenkins Security"
 section: security
 ---
 
+The Jenkins project takes security seriously.
+We make every possible effort to ensure users can adequately secure their automation infrastructure.
```

Comment:
I'm tempted to make this official policy for the `/security` URL space while I'm team lead :-)